cachemanager/handler_redis.go

```diff
@@ -0,0 +1,101 @@
+package cachemanager
+
+import (
+	"time"
+
+	"gopkg.in/redis.v3"
+	"strconv"
+)
+
+// NewHandlerRedis creates a redis handler
+func NewHandlerRedis(redisConfig *redis.Options, prefix string, ttl time.Duration) Handler {
```

Comment:
I would put `Addr string` directly in the function arguments instead of `redisConfig *redis.Options` so that the `redis.Options` is created within this function